agent/src/agent/api/routes/alerts.py

```diff
@@ -0,0 +1,187 @@
+import requests
+
+from copy import deepcopy
+from flask import Blueprint, request, jsonify
+from agent import destination
+from agent.api.forms.alerts import AlertsByStatusForm, AlertStatusForm
+from agent.destination.anodot_api_client import AnodotApiClient
+
+alerts = Blueprint('alerts', __name__)
+
+TRANSFORM_ALERT_GROUPS = {
```

Comment:
ok, but it will still be partly universal due to the specific filtering and transformations that are applied earlier

Reply:
yes, it will also help in case the JRC themselves will want to change the set of fields, this way they can do it themselves, no need for an additional release